WalletWasabi.Fluent/Helpers/TextHelpers.cs

```diff
@@ -9,7 +9,7 @@ namespace WalletWasabi.Fluent.Helpers;
 
 public static partial class TextHelpers
 {
-	public static string AddSIfPlural(int n) => n > 1 ? "s" : "";
+	public static string AddSIfPlural(int n) => n > 1 ? Lang.Resources.Utils_Plural : "";
```

Comment:
haha good try but this is wrong. 

person -> people
mujer -> mujeres
peccator -> pecatoribus
Mutter -> Mütter

Reply:
I changed to "Generic"